server/repository/src/migrations/v1_01_11/requisition.rs

```diff
@@ -12,5 +12,16 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
         "#
     )?;
 
+    // View to help figure out how many requisitions were done for a program and order type in a period
+
+    sql!(
+        connection,
+        r#"
+        CREATE VIEW requisitions_in_period AS
+        SELECT 'n/a' as id, program_id, period_id, store_id, order_type, count(*) as count FROM requisition
```

Comment:
Does this need to be only `REQUEST` Requistions? or at least add a filter for that?

Reply:
Added, thanks, this would have been really hard to find in a mid level warehouse that fulfils and requests order, great catch: a4a54acebafae15d88cfc3f0505889e1e83980f6